src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/FunctionsTests.cs

```diff
@@ -34,567 +34,568 @@ public void TestSignaturesV2_3_X()
 
             Column col = Column("col1");
```

Comment:
Let's do this one too.  Thanks for doing the others.

Reply:
have added a check for Column, still have `Column col = Column` as `col` is used later on